drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -4370,7 +4370,8 @@ void RasterizerSceneGLES3::render_shadow(RID p_light, RID p_shadow_atlas, int p_
 	RasterizerStorageGLES3::Light *light = storage->light_owner.getornull(light_instance->light);
 	ERR_FAIL_COND(!light);
 
-	uint32_t x, y, width, height, vp_height;
+	uint32_t x, y, width, height;
+	// uint32_t vp_height;
```

Comment:
Same here, the only references to `vp_height` are in the code you commented out, it's never used anywhere. Kill it.